I probably  misunderstood the `enable gpu` behavior here

```
19:07:18 processing relval_machine
19:07:18 processing relval_premix
19:07:18 Traceback (most recent call last):
19:07:18   File "/cvmfs/cms-ci.cern.ch/week1/cms-sw/cmssw/36529/21331/CMSSW_12_3_X_2021-12-16-1100/bin/slc7_amd64_gcc10/runTheMatrix.py", line 598, in <module>
19:07:18     ret = runSelected(opt)
19:07:18   File "/cvmfs/cms-ci.cern.ch/week1/cms-sw/cmssw/36529/21331/CMSSW_12_3_X_2021-12-16-1100/bin/slc7_amd64_gcc10/runTheMatrix.py", line 31, in runSelected
19:07:18     if len(undefSet)>0: raise ValueError('Undefined workflows: '+', '.join(map(str,list(undefSet))))
19:07:18 ValueError: Undefined workflows: 10824.502, 11634.502, 10824.506, 11634.506
```

--- a/RecoPixelVertexing/PixelTrackFitting/python/PixelTracks_cff.py
+++ b/RecoPixelVertexing/PixelTrackFitting/python/PixelTracks_cff.py
@@ -93,14 +93,17 @@ from Configuration.ProcessModifiers.pixelNtupletFit_cff import pixelNtupletFit
 
 from RecoPixelVertexing.PixelTriplets.pixelTracksCUDA_cfi import pixelTracksCUDA as _pixelTracksCUDA
 
+#Pixel tracks in SoA format on the CPU
+pixelTracksCPU = _pixelTracksCUDA.clone(
+    pixelRecHitSrc = "siPixelRecHitsPreSplitting",
+    idealConditions = False,
+    onGPU = False
+)
+
 # SwitchProducer providing the pixel tracks in SoA format on the CPU
 pixelTracksSoA = SwitchProducerCUDA(
     # build pixel ntuplets and pixel tracks in SoA format on the CPU
-    cpu = _pixelTracksCUDA.clone(
-        pixelRecHitSrc = "siPixelRecHitsPreSplittingSoA",
-        idealConditions = False,
-        onGPU = False
-    )
+    cpu = pixelTracksCPU
 )
 # use quality cuts tuned for Run 2 ideal conditions for all Run 3 workflows
 run3_common.toModify(pixelTracksSoA.cpu,
@@ -127,8 +130,11 @@ pixelNtupletFit.toReplaceWith(pixelTracksTask, cms.Task(
 ))
 
 
-# "Patatrack" sequence running on GPU
+# "Patatrack" sequence running on GPU (or CPU if not available)
 from Configuration.ProcessModifiers.gpu_cff import gpu
+(pixelNtupletFit & gpu).toModify(pixelTracksCPU,
+    pixelRecHitSrc = "siPixelRecHitsPreSplittingSoA",
+)
 
 # build the pixel ntuplets and pixel tracks in SoA format on the GPU
 pixelTracksCUDA = _pixelTracksCUDA.clone(

